lms/envs/common.py

```diff
@@ -1861,6 +1861,12 @@
     'provider.oauth2',
     'oauth2_provider',
 
+    # We don't use this directly (since we use OAuth2), but we need to install it anyway.
+    # When a user is deleted, Django queries all tables with a FK to the auth_user table,
+    # and since django-rest-framework-oauth imports this, it will try to access tables
+    # defined by oauth_provider.  If those tables don't exist, an error can occur.
+    'oauth_provider',
```

Comment:
@wedaly does the reverse migration for this look sane?  If we need to roll-back are there any dangers?